### ISSUE TITLE
No file path completion

### ISSUE THREAD

**Issue by lmas** [skelterjohn/go-pkg-complete#2]
Hey, I'm missing auto completion for local files in the current dir when running, for example, `go run <tab>`.

**Comment by thomasf** [comment]
The completion that used to be in the main go repo a while back has some additional features.. 

https://github.com/thomasf/dotfiles-thomasf-base/blob/d37483e017d9635b022b71cc84c1025c10f9cdce/.config-base/bash/completion.d/go

I hav looked around a bit and I don't think  it.s still  maintained somewhere else.

I guess people in general does not use multiple entries in GOPATH except for when doing non interactive stuff but the current implementation (in this repo) would break there since it does not split on `:`.. I've been thinking about just caching the `go list all` in an env var to speed up repeated completions.

I did switch to the completion in this repo yesterday but now I'm not sure since I miss the more complete completion support.

related CL https://codereview.appspot.com/105470043

Since the old bash completion seems to be orphaned(?) it's maybe a good idea to copy stuff from it so this repo can become the maintained version of more than just pkg completions? Even if it does not have a license header I'm assuming it's under the go license because of where it came from.

Sorry for getting a bit OT but I figured best to mention this before duplicating more work already done.
